pom.xml

```diff
@@ -76,10 +75,8 @@
 
     <dependencies>
         <dependency>
-            <!--Overriding spring-web version to remove vulnerability-->
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
```

Comment:
You could remove this dependency as it would be included in spring boot?

Reply:
removed now!